frontend/src/components/button/button.tsx

```diff
@@ -0,0 +1,68 @@
+import { type ReactNode } from 'react';
+
+type ButtonVariant = 'primary' | 'secondary' | 'tertiary';
+type ButtonSize = 'small' | 'medium';
+type ButtonType = 'button' | 'submit';
+
+type ButtonProperties = {
+    children: ReactNode;
```

Comment:
I guess there is no need to pass children because im pretty sure we would only need to pass the label

Reply:
This is for purpose of button with icon

Reply:
Then it would probably be better if you also add a label and make children optional since we won't need an icon every time

So it will remain like this
{label}
{children}

Reply:
Then it would probably be better if you also add a label and make children optional since we won't need an icon every time

So it will remain like this
{label}
{children}